src/features/defi/providers/yearn/components/YearnManager/Deposit/components/Status.tsx

```diff
@@ -77,69 +76,55 @@ export const Status = () => {
 
   if (!state) return null
 
-  const apy = state.opportunity?.metadata?.apy?.net_apy
-  const annualYieldCrypto = bnOrZero(state.deposit?.cryptoAmount).times(bnOrZero(apy))
-  const annualYieldFiat = annualYieldCrypto.times(marketData.price)
-
-  let statusIcon: React.ReactElement = <ArrowForwardIcon />
-  let statusText = StatusTextEnum.pending
-  if (state.deposit.txStatus === 'success') {
-    statusText = StatusTextEnum.success
-    statusIcon = <CheckIcon color='green' />
-  }
-  if (state.deposit.txStatus === 'failed') {
-    statusText = StatusTextEnum.failed
-    statusIcon = <CloseIcon color='red' />
-  }
+  const { statusIcon, statusText, statusBg, statusBody } = (() => {
+    let statusIcon: React.ReactElement = <AssetIcon size='xs' src={asset?.icon} />
+    let statusText = StatusTextEnum.pending
+    let statusBody = translate('modals.deposit.status.pending')
+    let statusBg = 'transparent'
+    if (state.deposit.txStatus === 'success') {
+      statusText = StatusTextEnum.success
+      statusIcon = <CheckIcon color='white' />
+      statusBody = translate('modals.deposit.status.success', {
+        opportunity: `${asset.name} Vault`,
+      })
+      statusBg = 'green.500'
+    }
+    if (state.deposit.txStatus === 'failed') {
+      statusText = StatusTextEnum.failed
+      statusIcon = <CloseIcon color='white' />
+      statusBody = translate('modals.deposit.status.failed')
+      statusBg = 'red.500'
+    }
+    return { statusIcon, statusText, statusBg, statusBody }
```

Comment:
We could avoid this mutation by switching on `state.deposit.txStatus` inside the IIFE.